File: c/resistor-color/src/resistor_color.c
#include "resistor_color.h"

int colorCode(resistor_band_t cc)
{
    return (int)cc;
}

resistor_band_t *colors()
{
    static resistor_band_t c[] = {
        BLACK, BROWN, RED, ORANGE, YELLOW,
        GREEN, BLUE, VIOLET, GREY, WHITE};
    return c;
}

File: csharp/resistor-color/ResistorColor.cs
using System;

public static class ResistorColor
{
    static string[] colors = new[] { "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white" };
    public static int ColorCode(string color) => Array.IndexOf(Colors(), color.ToLower());
    public static string[] Colors() => colors;
    
}